app/src/main-process/main.ts

```diff
@@ -72,6 +73,19 @@ function getExtraErrorContext(): Record<string, string> {
   }
 }
 
+const possibleProtocols = new Set(['x-github-client'])
+if (__DEV__) {
+  possibleProtocols.add('x-github-desktop-dev-auth')
+} else {
+  possibleProtocols.add('x-github-desktop-auth')
```

Comment:
Are we actually using `x-github-desktop-dev-auth` and `x-github-desktop-auth` yet? #1879 suggests that we intended to switch over to using them at some point, but for now afaict we're still just using `x-github-client`